### PR TITLE
more waits to stabilize test

### PR DESCRIPTION
**Pull request by edewit**
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

### CODE REVIEW

File: js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts [hunk 1]
```diff
@@ -45,16 +45,14 @@ describe("Realm settings general tab tests", () => {
...
     // Enable realm
     realmSettingsPage.toggleSwitch(`${realmName}-switch`);
-    masthead.checkNotificationMessage("Realm successfully updated", true);
+    masthead.checkNotificationMessage("Realm successfully updated");
+    sidebarPage.waitForPageLoad();
 
     // Disable realm
     realmSettingsPage.toggleSwitch(`${realmName}-switch`);
     realmSettingsPage.disableRealm();
     masthead.checkNotificationMessage("Realm successfully updated", true);
```
Comment (jonkoops) [inline js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts:54]: I think we can omit `true` here as it's the default.
```suggestion
    masthead.checkNotificationMessage("Realm successfully updated");
```

File: js/apps/admin-ui/cypress/downloads/downloads.html
Comment (jonkoops) [inline js/apps/admin-ui/cypress/downloads/downloads.html:1]: Not sure where this file is coming from, but pretty sure it's not supposed to be here.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts (hunk 1 above)
Comment (edewit) [reply js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts:54]: seems to not work

Comment (edewit) [reply js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts:54]: I had that before

Comment (jonkoops) [reply js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts:54]: Should not make a difference, as [it is the default value](https://github.com/keycloak/keycloak/blob/d5bf79b9329e81437d8f0100879e162efa7e2be2/js/apps/admin-ui/cypress/support/pages/admin-ui/Masthead.ts#L97), perhaps it's just flaky?

File: js/apps/admin-ui/cypress/downloads/downloads.html
Comment (jonkoops) [inline js/apps/admin-ui/cypress/downloads/downloads.html:1]: Interesting, wasn't aware this was checked in.

**Review by jonkoops — APPROVED** [review]

**Pull request merged**